Portfolio/index.html

```diff
@@ -7,7 +7,34 @@
   </head>
 
   <body style="margin: 0; padding: 0">
-    <!-- Write your code below for PORTFOLIO -->
-    <!-- to import image from public folder, do it like this= <img src="../public/css.icon.png" alt="CSS image"> -->
+    <h3>Deo</h3>
+    <ul>
+            <li>ABOUT</li>
+            <li>SKILLS</li>
+            <li>REVIEWS</li>
+            <li>PROJECT</li>            
```

Comment:
The image source path '../Portfolio/.public/jhonDoe.png' is incorrect. It should be '../Portfolio/public/jhonDoe.png'.